lib/resources/example_json/r4_examples_medication_request.rb

```diff
@@ -0,0 +1,2128 @@
+                                      "start": "2016-10-27T19:34:00.000-04:00"
+                                  }
+                              }
+                          }
+                      }
+                  ],
+                  "dispenseRequest": {
+                      "validityPeriod": {
+                          "start": "2016-10-27T19:34:51.000-04:00"
+                      }
+                  }
+              }
+          },
+          {
+              "fullUrl": "https://fhir-open.sandboxcerner.com/r4/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/MedicationRequest/63969329",
+              "resource": {
+                  "resourceType": "MedicationRequest",
+                  "id": "63969329",
+                  "meta": {
+                      "versionId": "0",
+                      "lastUpdated": "2016-10-27T23:37:50.000Z"
+                  },
+                  "text": {
+                      "status": "generated",
+                      "div": "&lt;div xmlns=\"http://www.w3.org/1999/xhtml\"&gt;&lt;p&gt;&lt;b&gt;Medication Request&lt;/b&gt;&lt;/p&gt;&lt;p&gt;&lt;b&gt;Status&lt;/b&gt;: Active&lt;/p&gt;&lt;p&gt;&lt;b&gt;Intent&lt;/b&gt;: Order&lt;/p&gt;&lt;p&gt;&lt;b&gt;Medication&lt;/b&gt;: Misc Prescription (Freetext)&lt;/p&gt;&lt;p&gt;&lt;b&gt;Patient&lt;/b&gt;: PETERS, TIMOTHY&lt;/p&gt;&lt;p&gt;&lt;b&gt;Authored On&lt;/b&gt;: Oct 27, 2016  6:37 P.M. CDT&lt;/p&gt;&lt;/div&gt;"
+                  },
+                  "status": "active",
+                  "intent": "order",
+                  "category": [
+                      {
+                          "coding": [
+                              {
+                                  "system": "http://terminology.hl7.org/CodeSystem/medicationrequest-category",
+                                  "code": "community",
+                                  "display": "Community",
+                                  "userSelected": false
+                              }
+                          ]
+                      },
+                      {
+                          "coding": [
+                              {
+                                  "system": "http://terminology.hl7.org/CodeSystem/medication-statement-category",
+                                  "code": "patientspecified",
+                                  "display": "Patient Specified",
+                                  "userSelected": false
+                              }
+                          ]
+                      }
+                  ],
+                  "reportedBoolean": true,
+                  "medicationCodeableConcept": {
+                      "text": "Misc Prescription (Freetext)"
+                  },
+                  "subject": {
+                      "reference": "Patient/1316024",
+                      "display": "PETERS, TIMOTHY"
+                  },
+                  "encounter": {
+                      "reference": "Encounter/6188499"
+                  },
+
```

Comment:
25 is way too many example med requests

Reply:
https://github.com/cerner/fhir.cerner.com/pull/332/commits/61f407d4385bd72a2b599551739c8e1fd09b3d67